drgn/helpers/linux/kconfig.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+"""
+Kconfig
+------
+
+
+The ``drgn.helpers.linux.kconfig`` module provides helpers for reading kernel
+configuration from memory. This is only supported if CONFIG_IKCONFIG is set.
+"""
+
+import gzip
+import types
+
+__all__ = (
+    "get_kconfig",
+)
+
+def get_kconfig(prog):
+    try:
+        return prog.cache["kconfig_options_map"]
+    except KeyError:
+        pass
+
+    try:
+        start = prog.symbol("kernel_config_data").address
+        size = prog.symbol("kernel_config_data_end").address - start
+        raw_data = prog.read(start, size)
+    except LookupError:
+        try:
```

Comment:
We should limit the `try` block to only contain the statement that we want to handle, i.e., `prog["kernel_config_data"]`.